Cargo.toml

```diff
@@ -20,6 +20,7 @@ appveyor = { repository = "serde-rs/json" }
 serde = { version = "1.0.60", default-features = false }
 indexmap = { version = "1.2", optional = true }
 itoa = { version = "0.4.3", default-features = false }
+lexical-core = { version = "0.7", optional = true, default_features = false, features = ["std", "correct", "format", "ryu"]}
```

Comment:
You can just use `default_features = true`, features = ["format"]`, since the default features are `std`, `ryu`, and `correct` always,

Reply:
Ok, thanks.